### PR TITLE
Fix the pretty_printing for print_schema.

### LINKED ISSUE #790: print-schema vs. compound primary keys
Just [saw](https://github.com/sgrif/crates.io/blob/5082c78938f2353e27c272e9eb089a8d1a62e9a1/src/schema.rs#L4-L5) this:

```rust
table! {
    badges (crate_id,
    badge_type) {
        crate_id -> Int4,
        badge_type -> Varchar,
        attributes -> Jsonb,
    }
}
```

The second like should be `badges (crate_id, badge_type) {`.

Looks like our [pretty printing](https://github.com/diesel-rs/diesel/blob/8c8c0ab9d1c877650ffcb804ac1ba6d96ed591b9/diesel_cli/src/pretty_printing.rs) code needs a few more test cases and one more special case.

### PR DESCRIPTION
**Pull request by Eijebong**
Do not insert a newline after a comma if we're between parenthesis.

Fixes #790

### CODE REVIEW

**Comment by killercup** [comment]
Looks good. Do you want to add a smaller unit test as well at the end of the pretty file?

**Comment by Eijebong** [comment]
Oh I missed that :o

**Comment by Eijebong** [comment]
Done

File: diesel_cli/tests/print_schema.rs [hunk 1]
```diff
@@ -181,3 +181,27 @@ table! {
...
+
+    // Make sure the project is setup
+    p.command("setup").run();
+
+    db.execute("CREATE TABLE asd (id INTEGER, qsd INTEGER, PRIMARY KEY (id, qsd));");
+
+    let result = p.command("print-schema").run();
+
+    assert!(result.is_success(), "Result was unsuccessful {:?}", result);
+    println!("{}", result.stdout());
```
Comment (Eijebong) [inline diesel_cli/tests/print_schema.rs]: Oups

**Comment by Eijebong** [comment]
Urg that deadlock is annoying >_<

**Pull request merged**